### ISSUE TITLE
Async loading of angular-sanitize with angular-loader draw a "Object #<Object> has no method 'extend' " error

### ISSUE THREAD

**Issue by ocolot** [angular/angular.js#6873]
I use the angular-seed where I load some more dependencies with `$script` as follow:

$script([
  '//ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js',
  '//ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular-route.min.js',
  '//ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular-resource.min.js',
  '//ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular-animate.min.js',
  '//ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular-sanitize.min.js',
  '//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.6.0/underscore-min.js',
  'js/app.js'
  ], function() {
...

I think angular-sanitize tries to call angular.extend before angular has been loaded. Does it sounds right?

**Comment by eperret** [comment]
I still see the issue with version 1.4.7 .  A little disappointed that this issue has been around for so long...

**Comment by lgalfaso** [comment]
It looks like this issue belongs to angular-seed as there is nothing angular specific with this issue. In angular-seed, the scripts and their dependencies should be loaded following `$script` conventions

**Comment by eperret** [comment]
@lgalfaso Can you point me to the angular-seed issue?  I would like to comment on it.

**Comment by lgalfaso** [comment]
@eperret there is no issue at angular-seed that I am aware to handle this issue. The issue reported here is simply unrelated to angular.
If you would like to contribute with a fix to angular-seed, feel free to create a PR.
